### DISCUSSION TITLE
useOpenAITTS - Lobe TTS

### DISCUSSION THREAD

**Discussion by giscus** [lobehub/lobe-ui#111] [Q&A]
# useOpenAITTS - Lobe TTS

https://tts.lobehub.com/components/use-open-aitts

**Comment by denisuoo** [comment]
The woman was at home feeding her dog 
When she heard someone knocking on her door from outside 
In the middle of the night which made her feel a bit scared 
Because her husband 
had died in a car accident a few days ago 
Leaving her and her unborn child alone 
No one would be looking for her at this time 
But the knocking still hadn't stopped 
Be brave and look at the cat's eye 
And yet it's pitch black outside 
She could not see the man's face clearly 
She quickly turned on the switch 
But to her surprise, it was still pitch black outside 
The woman quickly asked who it was 
Then came the voice of a stranger woman from outside 
She told the woman 
Her car broke down 
Wants to come in and make a call 
But the woman for her own safety 
How could she let a stranger in 
She turned down the offer with the reason that her husband was asleep 
But the stranger's next words made her hair stand on end
